presto-hive/src/main/java/io/prestosql/plugin/hive/HiveTableProperties.java

```diff
@@ -153,6 +155,7 @@ public HiveTableProperties(
                 stringProperty(TEXTFILE_FIELD_SEPARATOR, "TEXTFILE field separator character", null, false),
                 stringProperty(TEXTFILE_FIELD_SEPARATOR_ESCAPE, "TEXTFILE field separator escape character", null, false),
                 stringProperty(NULL_FORMAT_PROPERTY, "Serialization format for NULL value", null, false),
+                stringProperty(SPARK_TABLE_PROVIDER, "Internal Hive connector property", null, true),
```

Comment:
Do we want to add this here? It means users could set it directly

Reply:
I am aware. It's hidden. I see no other way to test this. Please advise.

Reply:
I thought I commented on the test code, but looks like that got lost. The test can use `getMetastoreClient()` to call the metastore directly to create the table, rather than going through `ConnectorMetadata`. This means we don't need to add the table property.

Reply:
Done